# HG changeset patch
# User Rik <rik@octave.org>
# Date 1349996806 25200
#      Thu Oct 11 16:06:46 2012 -0700
# Node ID b3762b89c3c0756fb011b302ef8dcf70328b4b40
# Parent  a7794a192b47c9143849c540010b200a16f33452
__init_fltk__.cc: Correct function name in gui_mode error message.

* __init_fltk__.cc(Fgui_mode): Correct function name in error message
when HAVE_FLTK is false.

diff --git a/libinterp/dldfcn/__init_fltk__.cc b/libinterp/dldfcn/__init_fltk__.cc
--- a/libinterp/dldfcn/__init_fltk__.cc
+++ b/libinterp/dldfcn/__init_fltk__.cc
@@ -2130,12 +2130,12 @@ This function is currently implemented o
         failed = true;
     }
 
   if (failed)
     error ("MODE must be one of the strings: \"2D\", \"3D\", or \"none\"");
 
   return octave_value (mode_str);
 #else
-  error ("mouse_wheel_zoom: not available without OpenGL and FLTK libraries");
+  error ("gui_mode: not available without OpenGL and FLTK libraries");
   return octave_value ();
 #endif
 }
